kubectl-plugin/pipeline/run.go

```diff
@@ -31,19 +32,42 @@ func newPipelineRunCmd(client dynamic.Interface) (cmd *cobra.Command) {
 	flags.StringVarP(&opt.namespace, "namespace", "n", "",
 		"The namespace of target Pipeline")
 	flags.BoolVarP(&opt.batch, "batch", "b", false, "Run pipeline as batch mode")
+	flags.StringArrayVarP(&opt.parameters, "parameter", "P", []string{}, "The parameters that you want to pass, example of single parameter: name=value")
```

Comment:
You can try this method to avoid parameter parsing.

`func (f *FlagSet) StringToStringP(name, shorthand string, value map[string]string, usage string) *map[string]string`

Reply:
Thanks for reminding, I will try it out.